tools/check_format.py

```diff
@@ -145,36 +149,35 @@ def checkBuildLine(line, file_path, reportError):
   if not whitelistedForProtobufDeps(file_path) and '"protobuf"' in line:
     reportError("unexpected direct external dependency on protobuf, use "
                 "//source/common/protobuf instead.")
-  if envoy_build_rule_check and '@envoy//' in line:
+  if envoy_build_rule_check and not isSkylarkFile(file_path) and '@envoy//' in line:
```

Comment:
@jmarantz do you remember why you've added this check for `@envoy//` and whether or not it should apply to `.bzl` files? My thinking is that we should retain `@envoy//` there, but I'm missing the context for why this check was added in the first place.

Reply:
It didn't work with our import process. Another option would be to change our import process.